python/lsst/analysis/tools/actions/vector/vectorActions.py

```diff
@@ -200,3 +231,27 @@ def __call__(self, data: KeyedData, **kwargs) -> Vector:
             correction = correction.to(u.mmag)
 
         return np.array(diff - correction.value)
+
+
+class PerGroupStatistic(VectorAction):
+    """Compute per-group statistic values."""
+
+    groupKey = Field[str](doc="Column key to use for forming groups", default="obj_index")
+    buildAction = ConfigurableActionField(doc="Action to build vector", default=LoadVector)
+    func = Field[str](doc="Name of function to be applied per group", default="count")
+
+    def getInputSchema(self) -> KeyedDataSchema:
+        return tuple(self.buildAction.getInputSchema()) + ((self.groupKey, Vector),)
+        #yield from self.buildAction.getInputSchema()
+        #yield (self.groupKey, Vector)
+
+    def __call__(self, data: KeyedData, **kwargs) -> Vector:
+
```

Comment:
odd spacing

Reply:
Good catch. It looks like that was a remnant of unfinished thought.